app/src/main/java/com/sdp13epfl2021/projmag/model/ImmutableProject.kt

```diff
@@ -80,6 +82,9 @@ data class ImmutableProject constructor(
                     "there are ${assigned.size} " +
                             "students currently assigned but only $nbParticipant allowed to work for the project"
                 )
+                //!sectionsManager.isListValid(allowedSections) -> Failure("a section in the section list doesn't exist $allowedSections")
+                 //!tagsManager.isListOfStringsValidTags(tags) -> Failure("a tag in the tag list doesn't exist $tags")
+
```

Comment:
Is this code commented because it breaks the projects already on Firebase ?

Reply:
Yes, for now we don't check this because most projects on firebase do not have the right tags and sections. This is a feature and not a bug